pkgs/by-name/co/cosmic-wallpapers/package.nix

```diff
@@ -8,12 +8,12 @@
 
 stdenvNoCC.mkDerivation (finalAttrs: {
   pname = "cosmic-wallpapers";
-  version = "1.0.0-alpha.5.1";
+  version = "1.0.0-alpha.6";
 
   src = fetchFromGitHub {
     owner = "pop-os";
     repo = "cosmic-wallpapers";
-    rev = "epoch-${finalAttrs.version}";
+    tag = "epoch-${finalAttrs.version}";
     forceFetchGit = true;
     fetchLFS = true;
     hash = "sha256-Exrps3DicL/G/g0kbSsCvoFhiJn1k3v8I09GhW7EwNM=";
```

Comment:
```suggestion
    hash = "";
```
Needs to be regenerated

Reply:
It doesn't need, it is the same commit

Reply:
![image](https://github.com/user-attachments/assets/5d8a2d07-d43e-4c88-a791-961bb1860c0c)

Reply:
Don't really see the point in updating then :shrug: